### PR TITLE
🎨 style: remove anniversary styling

### PR DESCRIPTION
**Pull request by larwaa**
## Proposed Changes

- Remove the custom anniversary styling as the anniversary draws to an end.

## To do

- [ ] Figure out what to do this component. Should it remain, but with updated content? Revert to the original design? Other suggestions? Personally, I really like it
Screen Shot 2022-03-10 at 00 49 43

## Types of Changes

- [ ] New feature
- [ ] Bug fix
- [ ] Enhancement/optimization
- [ ] Refactor
- [x] Style
- [ ] Build/dependencies
- [ ] Other

## Issue(s) fixed or closed by this PR

-

## Checklist

- [ ] I have added tests to cover my changes (for features/bugs)
- [x] I am pleased with the readability of the code (if not, state where you need input)
- [x] I have tested the changes and verified that they work and do not break anything (to the best of my ability)

### CODE REVIEW

File: frontend/src/components/navbar/Navbar.tsx [hunk 1]
```diff
@@ -32,7 +29,6 @@ const useStyles = makeStyles((theme) => ({
   },
   appBar: {
     background: theme.palette.primary.dark,
-    backgroundImage: "url('/static/anniversary/sparkles.gif')",
     backgroundPosition: "bottom",
```
Comment (mathiasraa) [inline frontend/src/components/navbar/Navbar.tsx:32]: backgroundPosition can be removed also :)

**Comment by mathiasraa** [comment]
I think the component should remain and just edit the content. Removing it makes the page seem kinda empty

**Comment by erikttv** [comment]
Perhaps upcoming events? And if there is a special event we want to promote, we can display it here? 

As of now we have no upcoming events, but maybe have "today's organization"? Where we present a random organization of the day and people can go in and read about that specific organization. The next day it changes to another organization.

**Comment by larwaa** [comment]
> Perhaps upcoming events? And if there is a special event we want to promote, we can display it here?
> 
> As of now we have no upcoming events, but maybe have "today's organization"? Where we present a random organization of the day and people can go in and read about that specific organization. The next day it changes to another organization.

I personally think "today's organization" is a great idea when there are no events to promote! Slight concern would be that a lot of the organisations are in a need of an update, hehe.

**Comment by hermannm** [comment]
This PR may become redundant with @mathiasraa's new styling overhaul.

**Pull request closed without merging**